src/Compilers/VisualBasic/Test/Emit/Attributes/AttributeTests_CallerArgumentExpression.vb

```diff
@@ -134,7 +134,7 @@ End Class
             Dim source As String = "
 Imports System
 Imports System.Runtime.CompilerServices
-Public Module Program
+Public Module Program2
```

Comment:
>Program2 [](http://example.com/codeflow?start=14&length=8)

Is this change necessary?

Reply:
Yes. We're compiling a C# program below (on line 194) that needs to call the `Log` method from this VB program.